core/custom_filters/custom_filter_wizard_storage.py

```diff
@@ -0,0 +1,118 @@
+import logging
+
+from collections import namedtuple
+from typing import List
+
+from core.custom_filters.custom_filter_registry_point import CustomFilterRegistryPoint
+from core.custom_filters import CustomFilterWizardInterface
+
+
+logger = logging.getLogger(__name__)
+
+
+class CustomFilterWizardStorage:
+    """
+        Class responsible for storing information on how to create filters based on the context of
+        a particular wizard creator. This hub provides information about the method for building
+        filters for a chosen field in specific object.
+
+        Constants:
+            __KEY_FOR_OBTAINING_CLASS: Indicates which element from the registry points should be
+                                       considered to retrieve information about the registered CustomFilterWizard class.
+            __FIELD: Identifies the first value of the final output tuple for building filters.
+            __FILTER: Identifies the second value of the final output tuple for building filters.
+            __VALUE: Identifies the third value of the final output tuple for building filters.
+    """
+
+    __KEY_FOR_OBTAINING_CLASS = 'class_reference'
+    __FIELD = 'field'
+    __FILTER = 'filter'
+    __VALUE = 'value'
+
+    @classmethod
+    def build_output_how_to_build_filter(cls, module_name: str, object_type: str, **kwargs) -> List[namedtuple]:
```

Comment:
I don't quite like this name, perhaps something like `build_custom_filters_definition` would be better.